regression/tests/enterprise/test_enterprise_enrollment_page.py

```diff
@@ -9,7 +9,7 @@
 
 class TestEnterpriseEnrollmentPage(EnterpriseTestBase):
     """
-    Test Enterprise Login
+    Test Enterprise Enrollment page
```

Comment:
Just for info that there are two enrollment pages in enterprise one is `enterprise course enrollment page` and other one is `enterprise program enrollment page`.

Since you have this class `TestEnterpriseEnrollmentPage` so I believe it will wrap all the tests related to both pages if not then I would suggest changing this class name to `TestEnterpriseCourseEnrollmentPage`.